embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceDeliveryCacheManager.kt

```diff
@@ -61,18 +61,18 @@ internal class EmbraceDeliveryCacheManager(
         snapshotType: SessionSnapshotType
     ): ByteArray? {
         return when (snapshotType) {
```

Comment:
There probably isn't enough decisions being made that checks `SnapshotType` to formalize them as properties of the enum?